Dockerfile

```
@@ -0,0 +1,23 @@
+# Use a Rust base image
+FROM rust:latest
+
+# Install necessary tools (curl and tar)
+RUN apt-get update && apt-get install -y curl tar
+
+# Install mdBook
+RUN cargo install mdbook
+
+# Install mdbook-admonish
+RUN cargo install mdbook-admonish
+
+# Download and extract mdbook-discord-components
+WORKDIR /usr/local/bin
+RUN curl -L -o components.tar.gz https://github.com/NilPointer-Software/mdbook-discord-components/releases/download/v0.2.0/mdbook-discord-components-v0.2.0.tar.gz \
+    && tar xzf components.tar.gz \
+    && rm components.tar.gz
```

You shouldn't download files to `/usr/local/bin`. \
It would be better to extract the tar archive to that directory instead
```suggestion
RUN curl -L -o components.tar.gz https://github.com/NilPointer-Software/mdbook-discord-components/releases/download/v0.2.0/mdbook-discord-components-v0.2.0.tar.gz \
    && tar xzf components.tar.gz -C /usr/local/bin \
    && rm components.tar.gz
```